### DISCUSSION TITLE
oracle11g、flink1.18.0、cdc3.0.1启动任务要等大概20分钟才会有数据进来

### DISCUSSION THREAD

**Discussion by lfift** [apache/flink-cdc#3086] [Q&A] [answered]
配置了：
log.mining.strategy和log.mining.continuous.mine也没有用
![image](https://github.com/ververica/flink-cdc-connectors/assets/31279478/591e8e6e-64e1-475c-8540-f483ac047c57)

**Comment by supercena** [comment]
我感觉 oracle 版本的 ，社区都还没有精力维护

**Reply by lfift** [reply]
大家都用mysql，主要oracle是闭源的

**Comment by lfift** [comment, marked as answer]
已解决，原因是默认情况会为除系统表以外的其他表捕获表结构，表越多就越慢。不知道为什么会这样。通过增加配置database-history-store-only-captured-tables-ddl=true，就只会捕获指定的表

**Reply by lfift** [reply]
这个属性官方说配置为true 后可能会丢失数据，这个还没搞明白
